test/functionalTest/cases/117_convop_using_standard_ops/putAvailabilitySubscriptionConvOp.test

```diff
@@ -0,0 +1,226 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+ConvOp putAvailabilitySubscriptionConvOp: PUT /v1/registry/contextAvailabilitySubscriptions/{subscriptionId}
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
```

Comment:
I was thinking about that too but as the purpose of this test is not to test the subscriptions themselves but the ability to create/modify them. We have to assume that if the broker says "ALL OK", all is ok :-)

What I would have liked here is to ask the broker for a listing of its subscriptions.